lib/src/main/java/com/futurewei/alcor/common/rbac/StrictRbacManager.java

```diff
@@ -0,0 +1,232 @@
+/*
+ *
+ * Copyright 2019 The Alcor Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *         you may not use this file except in compliance with the License.
+ *         You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *         Unless required by applicable law or agreed to in writing, software
+ *         distributed under the License is distributed on an "AS IS" BASIS,
+ *         WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *         See the License for the specific language governing permissions and
+ *         limitations under the License.
+ * /
+ */
+
+package com.futurewei.alcor.common.rbac;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.entity.*;
+import com.futurewei.alcor.common.exception.*;
+import com.futurewei.alcor.common.utils.ControllerUtil;
+import com.futurewei.alcor.common.utils.JsonUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Field;
+import java.util.*;
+
+public class StrictRbacManager implements RbacManger {
+    private static final Logger LOG = LoggerFactory.getLogger(StrictRbacManager.class);
```

Comment:
We have also a Alcor logger under common lib. do you find anything missing compared to slf4j? If not, maybe a good idea to switch to our own logger :-)

Reply:
ok

Reply:
> We have also a Alcor logger under common lib. do you find anything missing compared to slf4j? If not, maybe a good idea to switch to our own logger :-)

Slf4j is a stateful and strong log frame. All our own logger features can supported by Slf4j.